src/Evolution/Systems/GrMhd/GhValenciaDivClean/BoundaryConditions/Factory.hpp

```diff
@@ -62,7 +62,6 @@ using standard_boundary_conditions = tmpl::push_back<
 
 /// Boundary conditions that work with finite difference.
 using standard_fd_boundary_conditions =
-    tmpl::list<ConstraintPreservingFreeOutflow, DirichletAnalytic,
```

Comment:
Why do we need to change the definition of standard? Please keep as-is

Reply:
As mentioned earlier the DirichletAnalytic boundary condition doesn't work with numeric initial data. Since the `standard_fd_boundary_conditions` alias is only used in the executable and in tests I removed DirichletAnalytic here and appended it to the list where it's needed.